### DISCUSSION TITLE
是否可以支持增加MOD是否默认开启的功能

### DISCUSSION THREAD

**Discussion by RiverSDream** [Meloong-Git/PCL#816] [提问] [answered]
一个小的更新
我一直饱受这个问题的困扰，因为每次提前下载好或者拖入一个我暂时不需要用的MOD之后，都要到一大堆MOD列表中去关闭它，希望可以随时切换

**Comment by WForst-Breeze** [comment, marked as answer]
既然你暂时不需要，其实你没有必要将其下载下来或者拖入 mods 文件夹啊（）况且现在的mod列表支持搜索mod，手动关闭也很方便的🤔

补充：请见下方讨论，不考虑制作此功能orz

**Comment by zkitefly** [comment]
我觉得你的想法是这样的
加载模组开关配置文件，当想切换至配置1时，直接点击切换，PCL自动帮你关闭和开启相对应名字的模组
似乎还不错，不过我觉得你准备两个版本分别放不同的模组不是更好？

**Reply by WForst-Breeze** [reply]
hmmm，这么一说也的确，但这样感觉和 #818 提及的第一条有点像呢🤔

**Reply by WForst-Breeze** [reply]
@LTCatt 您考虑一下这个功能怎么样？🤔技术上的东西我也不是很懂…… 麻烦龙猫了（

**Reply by LTCatt** [reply]
实际上是和 #818 类似……基本不会考虑做的 Orz
